scripts/zkai_gkr_d128_projection_scaling_preflight_gate.py

```
@@ -0,0 +1,669 @@
+    expect_equal(loaded["minimal"].get("schema"), "zkai-minimal-transformer-block-benchmark-v1", "minimal schema")
+    expect_equal(loaded["selector"].get("schema"), "zkai-hybrid-proof-pressure-selector-v1", "selector schema")
+    expect_equal(loaded["tablero"].get("schema"), "zkai-tablero-hybrid-zkml-boundary-v1", "Tablero schema")
+    loaded["raw"] = raws  # type: ignore[assignment]
+    return loaded
+
+
+def result_by_fixture(results: list[Any], fixture: str) -> dict[str, Any]:
+    for row in results:
+        if isinstance(row, dict) and row.get("fixture") == fixture:
+            return row
+    raise GkrProjectionPreflightError(f"fixture missing: {fixture}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)